src/__tests__/IngridientsList.test.js

```diff
@@ -0,0 +1,40 @@
+import {
+  render,
+  fireEvent,
+  waitFor,
+  screen,
+  getByTestId,
+  within,
+  getAllByTestId,
+  queryAllByTestId,
+} from "@testing-library/react";
+import "@testing-library/jest-dom";
+import IngridientsList from "../components/IngridientsList";
+import userEvent from "@testing-library/user-event";
+
+describe("IngridientsList", () => {
+  test("Render ingridients list", async () => {
+    const list = ["cheese", "lemon"];
+    const { queryByText } = render(<IngridientsList ingridients={list} />);
+
+    expect(queryByText("cheese")).not.toEqual(null);
+    expect(queryByText("lemon")).not.toEqual(null);
+  });
+});
+
+describe("IngridientsList", () => {
+  test("Delete ingridient", async () => {
```

Comment:
Nit. I would suggest more descriptive naming: "Should call `handleDelete` when user click clear icon"